src/main/java/com/xceptance/xlt/report/providers/BasicTimerDataProcessor.java

```diff
@@ -121,6 +121,11 @@ public TimerReport createTimerReport(final boolean generateHistogram)
         // set the counts
         final double count = runTimeStatistics.getCount();
         final long duration = Math.max((getEndTime() - getStartTime()) / 1000, 1);
+        
+        if (count != 0)
+        {
+            timerReport.errorPercentage = ReportUtils.convertToBigDecimal(100 * totalErrors / count);
+        }
```

Comment:
Elsewhere you set the percentage to 0 in case the count is 0. Do the same here as well?

Reply:
Does not really matter because it is implicitly set to 0 if it was never specified. I can change it if you think it's better (readable?) that way.

Reply:
Please set it to 0. `errorPercentage` is a BigDecimal and would otherwise either be null or missing in the XML, not sure.